shared/resources/src/main/res/values/strings.xml

```diff
@@ -170,15 +170,15 @@
     <string name="sync_data_ivy_cloud">SYNC YOUR DATA ON THE IVY CLOUD</string>
     <string name="data_integrity_protection_warning">Data integrity and protection aren\'t guaranteed!</string>
     <string name="or_enter_with_offline_account">OR ENTER WITH OFFLINE ACCOUNT</string>
-    <string name="offline_warning">Your data will be saved locally (only on your phone) and won\'t be synced with the cloud. You risk losing it if you uninstall the app or change your device. You can always activate sync later if you decide to.</string>
+    <string name="offline_warning">Your data will be saved locally only and won't be synced with the cloud. Uninstalling the app or changing your device could result in data loss. You can enable sync later if needed.</string>
```

Comment:
not escaping `'` with `\'` may crash the app - please do these updates in Android Studio